source/docs/articles/sites/domains/mobile-redirection.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Enable Mobile Redirection with CloudFlare
+description: Learn how to use CloudFlare to set up mobile redirection.
+category:
+  - developing
+keywords: dns, cloudflare, mobile, redirect-incoming-requests, mobile tools
+---
+This article walks you though setting up a mobile redirect service through CloudFlare to automatically redirect visitors using mobile device visitors to a mobile optimized home page.
+
+## Before You Begin
+
+You'll need to have:  
+- A [CloudFlare account](https://www.cloudflare.com/a/sign-up)  
```

Comment:
Suggest clarifying assumed config like a domain added to the live environment with DNS already configured using CloudFlare as a provider. Also, an already built and optimized mobile home page.